api/src/wfl/module/covid.clj

```diff
@@ -96,27 +97,40 @@
 ;  ""
 ;  [])
 
+
 (defn verify-source!
+  "Verify that the `source` is named Terra DataRepo.
```

Comment:
I know you've been working iteratively but when you reach a stopping point, you could look at your docstrings and see if they can be condensed to summarize your operation rather than stepping through each line.

Ex. "Verify that the `source` is named Terra DataRepo` is clearly understood by reading your code.